.travis.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       addons:
         snaps:
           - name: snapcraft
-            channel: edge
+            channel: stable
```

Comment:
candidate at least :-)

Reply:
The next step will be to build snapcraft from the resulting snap.  Any reason not to build from stable as the starting point? Seems to me that should always work...